uefi/src/helpers/logger.rs

```diff
@@ -107,30 +107,17 @@ impl log::Log for Logger {
 
         if !output.is_null() {
             let writer = unsafe { &mut *output };
-            let result = DecoratedLog::write(
+
+            // Ignore all errors. Since we're in the logger implementation we
+            // can't log the error. We also don't want to panic, since logging
+            // is generally not critical functionality.
+            let _ = DecoratedLog::write(
```

Comment:
nit: We could write it to the serial device and the [debugcon](https://phip1611.de/blog/how-to-use-qemus-debugcon-feature/) device in that case.